### PR TITLE
cleaning some preview and deleting the duplicate list 

### PR DESCRIPTION
**Pull request by skyrag**
this pr is about cleaning and debugging the inventory screen, it also implement the tests needed for the screen so that we have the coverage needed

### CODE REVIEW

**Review by Mass-14 — CHANGES_REQUESTED** [review]
image

The item number of each field brakes the coherence in the UI when it has 2 digits (which go in column and not on the same line). Also this item number field is clickable, it shouldn't, and finally it should have the same padding with respect with the right edge, as the textfields.

ItemUi.kt : 
- the value Date (line 45) shouldn't be with a capital D
- the value time (line 51) in unused, it should be either commented with a explanation of the potential use, or deleted.

ItemList.kt : I just learned that the modifier argument of a composable function should be the first of the arguments. It's not a big deal but wanted to share this :)

InventoryScreen.kt : unused import line 22

InventoryViewModel.kt : 
- the private function getItems() isn't used, it should be commented (if potentially useful in the future)
- value users is unused (line 164), commented or deleted
- lots of non conventional naming (camelCase not respected, follow your IDE recommendations) 

Otherwise functional correctly, good job !

**Comment by skyrag** [comment]
thanks for the feedback, concerning the item number it doesnt appear like this on my phone so, i'm surprised but i'll look into it and concerning the fact that it is clickable it is suppose to be like that because for the loan part of the app we want to have a clickable text. Instead of the number of items it would be a see all and because we use the same function to show both it is just a clickable that does nothing. But if you have another idea on how to do it, we can talk about it

**Comment by Mass-14** [comment]
In order for this field to be clickable only when needed (see all), the clean solution would be to have an additional parameter in the ItemListColumn composable : `clickableCorner: Boolean = false,` with a ternary operator on the modifier (line 63) to give the onClickCorner action only if clickableCorner is True.

**Review by Mass-14 — CHANGES_REQUESTED** [review]
ItemUi.kt :
- unused import (line 16)

ItemListSeparator : 
- misleading `isClickable` parameter name, it should be contain at least the word _clickable_ and _corner_, since there is onClick and onClickCorner.
- additionally some the naming convention isn't respected in two parameters (line 35 and 38)
- the JavaDoc needs to be updated

InventoryViewModel :
- the update function (line 111) isn't called anmyore

InventoryScreen :
- line 106 and 121, the TODO should be removed since this is going to remain empty

File: app/src/main/java/com/android/partagix/ui/screens/InventoryScreen.kt
Comment (Mass-14) [inline app/src/main/java/com/android/partagix/ui/screens/InventoryScreen.kt:1]: The line 73 exceeds the 80 chars width limit, it should be split on two lines.

File: app/src/main/java/com/android/partagix/ui/components/ItemList.kt [hunk 1]
```diff
@@ -33,9 +34,11 @@ fun ItemList(
     users: List<User>,
     loan: List<Loan>,
     onClick: (Item) -> Unit,
-    modifier: Modifier = Modifier,
     stickyHeader: @Composable() (() -> Unit)? = null,
 ) {
+  println(itemList.size)
```
Comment (Mass-14) [inline app/src/main/java/com/android/partagix/ui/components/ItemList.kt:39]: line 39,40,41 : warning you forgot some println

File: app/src/main/java/com/android/partagix/model/InventoryViewModel.kt
Comment (Mass-14) [inline app/src/main/java/com/android/partagix/model/InventoryViewModel.kt:1]: lots of comments not relevant anymore, and prints that should be deleted

File: app/src/androidTest/java/com/android/partagix/inventory/InventoryTest.kt
Comment (Mass-14) [inline app/src/androidTest/java/com/android/partagix/inventory/InventoryTest.kt:1]: Empty test on line 96

**Review by Mass-14 — COMMENTED** [review]
All changes are minor, overall great work !

**Review by Mass-14 — DISMISSED** [review]

**Review by Patrick-Pataky — APPROVED** [review]
Great job ! It's finally time to merge it :)

**Pull request merged**